tests/test_rebase_command.rs

```diff
@@ -193,8 +193,8 @@ fn test_rebase_single_revision() {
     let stdout = test_env.jj_cmd_success(&repo_path, &["rebase", "-r", "b", "-d", "a"]);
     insta::assert_snapshot!(stdout, @r###"
     Also rebased 3 descendant commits onto parent of rebased commit
-    Working copy now at: b5ce312dacd3 (no description set)
-    Added 0 files, modified 0 files, removed 2 files
+    Working copy now at: e7299ad0c9a7 (no description set)
+    Added 0 files, modified 0 files, removed 1 files
```

Comment:
I'm not sure what changed here to affect the number of files, or whether it's a problem.

I'm less worried about the working copy id, since maybe the parents got reordered.

Reply:
(See my next comment)